inc/saiipmcgroup.h

```diff
@@ -93,11 +93,24 @@ typedef enum _sai_ipmc_group_member_attr_t
      * @brief IPMC output id
      *
      * @type sai_object_id_t
-     * @flags MANDATORY_ON_CREATE | CREATE_ONLY
+     * @flags CREATE_ONLY
```

Comment:
This should be discussed and agreed in community.

Now we are allowing a a NULL output ID for normal IPMC processing. Earlier this constraint was handled in orchagent and now SAI adapter have to handle it.

Reply:
Took care of this